students/elaine_x/session03/list_lab_ex.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+'''
+##########################
+#Python 210
+#Session 03 - List Lab
+#Elaine Xu
+#Jan 28,2019
+###########################
+'''
+
+############# Series 1 ################################
+print("Series 1\nList of Fruit:")
+FRUITS = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(FRUITS)
+
+#Ask the user for another fruit and add it to the end of the list
+FRUIT = input('Enter another fruit: ')
+FRUITS.append(FRUIT)
+print(FRUITS)
+
+#Ask user for a number, display the number and the fruit corresponding to the number
+NUMBER = int(input('Enter an index number (1 is first): '))
+print("The "+ str(NUMBER) +"th fruit is "+FRUITS[NUMBER-1])
+
+#Add another fruit to the beginning of the list using “+” and display the list
+print()
+print('Adding "Kiwi" to the beginning of the list')
+FRUITS = ['Kiwi'] + FRUITS
+print(FRUITS)
+
+#Add another fruit to the beginning of the list using insert() and display the list
+print()
+print('Adding "Banana" to the beginning of the list')
+FRUITS.insert(0, 'Banana')
+print(FRUITS)
+
+#Display all the fruits that begin with “P”, using a for loop
+print()
+print('Display all the fruits that begin with "P"')
+for item in FRUITS:
+    if item[0] == "P":
+        print(item)
+
+
+############# Series 2 ################################
+print()
+print("Series 2\nList of Fruit:")
+FRUITS2 = FRUITS[:]
+print(FRUITS2)
+
+#Remove the last fruit from the list
+print()
+FRUITS2.pop()
+print("Deleting the last fruit:")
+print(FRUITS2)
+
+#Ask the user for a fruit to delete, find it and delete it
+print()
+FRUIT_DELETE = input("Enter a fruit to delete: ")
+FRUITS2.remove(FRUIT_DELETE)
+print(FRUITS2)
+
+#Bonus: List*2. Keep asking until a match is found. Once found, delete all occurrences
+print()
+FRUITS2B = FRUITS[:]*2
+print('List * 2:', FRUITS2B)
+FRUIT_DELETE2 = input("Bonus: Enter a fruit to delete: ")
+while FRUIT_DELETE2 in FRUITS2B:
+    FRUITS2B.remove(FRUIT_DELETE2)
+print(FRUITS2B)
+
+
+############# Series 3 ################################
+print()
+print("Series 3\nList of Fruit:")
+FRUITS3 = FRUITS[:]
+print(FRUITS3)
+FRUITS3_COPY = FRUITS3[:]
+
+for fruit in FRUITS3:
+    choice = ''
+    while choice.lower() != 'yes' and choice.lower() != 'no':
+        choice = input("Do you like " + fruit.lower() + "? (yes or no):")
```

Comment:
if they just enter `blah` it will move on to the next fruit, how can you avoid that?

Reply:
while choice.lower() != 'yes' and choice.lower() != 'no':
The while loop will keep looping until the input is 'yes' or 'no' (upper and lower case).

Reply:
lol, yep your right, read it wrong.